"0","assocstats(table(vax_cont_all99$vax_cat2,vax_cont_all99$lca4_full))"
"1","                "
"1","    X^2"
"1"," df"
"1"," P(> X^2)"
"1","
Likelihood Ratio"
"1"," 244.73"
"1","  9"
"1","        0"
"1","
Pearson         "
"1"," 278.02"
"1","  9"
"1","        0"
"1","
"
"1","
"
"1","Phi-Coefficient   :"
"1"," "
"1","NA"
"1"," "
"1","
"
"1","Contingency Coeff.:"
"1"," "
"1","0.193"
"1"," "
"1","
"
"1","Cramer's V        :"
"1"," "
"1","0.113"
"1"," "
"1","
"
"0","vax_cont_all99%>%"
"0","  group_by(round)%>%"
"0","  mutate(vax_cat2 = as.numeric(vax_cat2),"
"0","         lca4_full = as.numeric(lca4_full))%>%"
"0","  summarise("
"0","    correlation = list(cor.test(vax_cat2, lca4_full, method = ""spearman"")))%>%"
"0","  mutate("
"0","    cor_value = map_dbl(correlation, ~ .$estimate),"
"0","    p_value = map_dbl(correlation, ~ .$p.value)"
"0","  ) %>%"
"0","  select(-correlation)"
"2","Warning: [38;5;232mThere were 3 warnings in `summarise()`.
The first warning was:[39m
[38;5;232m[36mâ„¹[38;5;232m In argument: `correlation = list(cor.test(vax_cat2, lca4_full, method = ""spearman""))`.
[36mâ„¹[38;5;232m In group 1: `round = ""r2""`.[39m
Caused by warning in `cor.test.default()`:
[33m![39m Cannot compute exact p-value with ties
[38;5;232m[36mâ„¹[38;5;232m Run ]8;;ide:run:dplyr::last_dplyr_warnings()dplyr::last_dplyr_warnings()]8;; to see the 2 remaining warnings.[39m"
